cmd/srcd/cmd/errors.go

```diff
@@ -1,16 +1,16 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/src-d/engine/docker"
-
-	"github.com/sirupsen/logrus"
 )
 
-// fatal converts known errors to human friendly message and logs it with fatal level
-func fatal(err error, format string, args ...interface{}) {
-	msg := fmt.Sprintf(format, args...)
+// wrapHumanizeErr wraps and converts known errors to human friendly message
```

Comment:
wrong name after the function rename.

Reply:
Fixed [here](https://github.com/src-d/engine/pull/359/commits/9d880dd2ce739939edd1d8c63d109a9d47664474).